### ISSUE TITLE
CTRL+ALT+Up is intercepted by some window managers

### ISSUE THREAD

**Issue by turt2live** [vector-im/element-web#16332]
linux mint cinnamon, at least

**Comment by ghosty-be** [comment]
linux mint cinnamon has the following keybindings overlapping:
keyboard -> shortcuts -> general -> 
"show the workspace selection screen" is bound to ctrl+alt+up
"show the window selection screen" is bound to ctrl+alt+down
(for now replaced up and down by pgup and pgdown so it does not interfere with element)

**Comment by SimonBrandner** [comment]
I like the idea of avoiding these overlaps but I am not sure we can always avoid them... I see configurable shortcuts as the only solution which will actually solve the problem, otherwise, we might run into another overlap. What's the dev team's view on this?

**Comment by t3chguy** [comment]
Configurable shortcuts is definitely desired, I don't think its on the immediate roadmap however

**Comment by ghosty-be** [comment]
yeah configurable shortcuts would be awesome as I could remap whole thing to work more irc style then too :) 
(just use up and down instead of ctrl + alt + up or down for example) 
The current up or down shortcuts for example are actions I rarely need... repeating a command however (especially when working with bots, trying to fix a typo) are far more relevant...

**Comment by SaschaCowley** [comment]
`Ctrl` + `Alt` + `<arrow>` also interferes with table navigation in NVDA (screen reader).

**Comment by c7hm4r** [comment]
From my intuition, `Ctrl` + `Alt` + … shortcuts refer to functions of the OS or the window manager.

**Comment by c7hm4r** [comment]
What about `Ctrl` + `<arrow>`?

**Comment by t3chguy** [comment]
That's a standard text navigation shortcut https://www.howtogeek.com/115664/42-text-editing-keyboard-shortcuts-that-work-almost-everywhere/

![image](https://user-images.githubusercontent.com/2403652/125925345-c430031d-1a1c-4c13-9292-80b39651e51f.png)

**Comment by c7hm4r** [comment]
Oh, good to know. Though, will Element ever have the concept of editing paragraphs?

**Comment by c7hm4r** [comment]
Related: #10733
